src/callback.ts

```diff
@@ -48,14 +49,15 @@ export default function callback(clientProvider: IOidcClientProvider, settings:
 
     // Create the session.
     const sealed = await Iron.seal(session, settings.session.cookieSecret, Iron.defaults);
-    setCookie(nextContext, 'oidc:session', sealed, {
+    res.setHeader('Set-Cookie', serialize('oidc:session', sealed, {
```

Comment:
I think to be safe we should probably also set the domain here? We want to avoid cookies to be set on now.sh and instead have them applied to acme-website.now.sh instead.